### ISSUE TITLE
[Request] Adobe block textures

### ISSUE THREAD

**Issue by Dzurky** [Alsender/Earthworks#4]
Heya, I see that your mod is growing which is fantastic to see and now that it's got a config file, I wanted to ask whether it would be possible to add an option that disables the random spots on Adobe blocks. I really love these blocks however I am not a really big fan of the random spotting so if it was possible to add this as an option in the config file it would be much appreciated.

**Comment by Alsender** [comment]
I have no idea if how to start about that.  I'll have a poke around, looking at some other sources to see if something like that can be done, but I make no promises...

**Comment by Barhandar** [comment]
The random spots also mean if you C&B the adobe, it will 1: all have the same texture variant and 2: periodically switch variant.  Which does not look good and results in unexpected change of looks of c-bed blocks, respectively.

**Comment by Alsender** [comment]
Alright, so I cannot find an example of a block changing texture on a config option being ticked.  I've found examples of tile entities changing textures like that, but I've also heard tell of tile entities disappearing in 1.12.  Sorry to say, but I'm not going to be able to make this request a reality.

**Comment by Barhandar** [comment]
>Alright, so I cannot find an example of a block changing texture on a config option being ticked.

https://github.com/Vazkii/Quark/blob/master/src/main/java/vazkii/quark/tweaks/feature/BetterVanillaTextures.java

I think this could be used to override adobe blockstates or their textures.

**Comment by Alsender** [comment]
That might be of some use.  Vazkii's code is always drawn out and somewhat hard to follow, but I'll see what this leads to
